react-components/src/data-providers/core-dm-provider/filterNodesByMappedTo3d.ts

```diff
@@ -46,11 +51,12 @@ export async function filterNodesByMappedTo3d(
     return {
       view: viewWithNodes.view,
       instances: viewWithNodes.instances.filter((instance) => {
-        if (!isDmsInstance(instance.properties[spaceFromView]?.[assetExternalIdWithVersion]?.object3D)) {
+        if (
+          !isDmsInstance(instance.properties[spaceFromView]?.[assetExternalIdWithVersion]?.object3D)
+        ) {
           return false;
         }
-        const object3dId =
-          instance.properties[spaceFromView][assetExternalIdWithVersion]?.object3D;
+        const object3dId = instance.properties[spaceFromView][assetExternalIdWithVersion]?.object3D;
         if (!isString(object3dId.externalId) || !isString(object3dId.space)) {
           return false;
         }
```

Comment:
Isn't the top-most condition here the only one we need? The other check seems unnecessary

Reply:
Agreed, fixed in [631c932](https://github.com/cognitedata/reveal/pull/5078/commits/631c932c7312c6234aeba0ecf06dc57fd32c012b)